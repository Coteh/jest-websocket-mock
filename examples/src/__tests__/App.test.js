import React from "react";
import renderer from "react-test-renderer";
import App from "../App";

jest.mock("../Messages", () => "Messages");
jest.mock("../MessageInput", () => "MessageInput");

describe("The App component", () => {
  it("renders the app skeleton", () => {
    const component = renderer.create(<App />);
    expect(component).toMatchSnapshot();
  });
});
