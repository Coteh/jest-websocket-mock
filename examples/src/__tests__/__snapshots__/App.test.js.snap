// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The App component renders the app skeleton 1`] = `
<div
  className="App"
>
  <ConnectionIndicator />
  <Messages />
  <MessageInput />
</div>
`;
